tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"compilerOptions": {
 		"baseUrl": "./",
-		"target": "ES2019",
+		"target": "ES2021",
```

Comment:
This was previously blocked by webpack 4 not supporting es2021 syntaxThis was previously blocked by webpack 4 not supporting es2021 syntax